### PR TITLE
Need help testing this.

### PR DESCRIPTION
**Pull request by PythoniCode**
Edit:

I've been doing some testing and tweaking of my own and will update this because as of now these changes do not work for Multiple CQ.

(Sorry I am new to GitHub lemme know if I am fucking shit up)

### CODE REVIEW

**Comment by Tune389** [comment]
Thanks for the request I test this out this weekend and will merge it.

File: TapTitans2AdvancedClickBot.txt [hunk 1]
```diff
@@ -500,11 +521,11 @@ goto :clanQuestClose
 	endif
 	
 :clanQuestCloseCheck
-	if #clanQuestRunsPerReadyUp == #clanQuestCount
+	if #recheck = 1
```
Comment (Tune389) [inline TapTitans2AdvancedClickBot.txt]: recheck = 1 dont't work -> change it to "=="

File: TapTitans2AdvancedClickBot.txt [hunk 2]
```diff
@@ -441,23 +443,42 @@ goto :start
...
-	getColor #color1 175 335
+	getRGB #colorRed #colorGreen #colorBlue 175 335
 	
 	#clanQuestLoopCount = 0
+
+	getRGB #colorRed2 #colorGreen2 #colorBlue2 295 712
 	
-	//addidional quest
-	if #color1 == 2055160
+	//addidional quest (MY EDIT: Make sure that it has a wider scope of possible colors to make sure it does the quest)
```
Comment (Tune389) [inline TapTitans2AdvancedClickBot.txt]: Can't understand that massive checks. Checking the boss progress bar should be enough here? Biggest Problems are the fading screens and the different loading times for max speed.

File: TapTitans2AdvancedClickBot.txt [hunk 3]
```diff
@@ -441,23 +443,42 @@ goto :start
...
+	if #colorRed > 240 and #colorRed < 255 and #colorRed2 > 90 and #colorRed2 < 110 and #clanQuestRunsPerReadyUp > 0
```
Comment (Tune389) [inline TapTitans2AdvancedClickBot.txt]: Maybe use full to prevent dia drain after 1 hour.

**Comment by PythoniCode** [comment]
Thanks for all the feedback !

I am trying a different method to check for multiple CQ that distinguished the buttons for using 5 25 etc. diamonds.

I see how yours works better though and I will make some changes to my code.

I hope it helps get the new version out though! Multiple CQ is in popular demand.

**Pull request closed without merging**